scenarios/httpclient.benchmarks.yml

```diff
@@ -47,6 +80,7 @@ scenarios:
         # Leaving the copied values unless the way to substitute correctly is found (other configs seem to employ copying strategy)
         # Overriding via console arg (e.g. `--variable serverPort=4321`) will work for *both* global and job's variables
         serverPort: 5000
+        presetHeaders: '{{requestHeaders}}' # TODO: unsupported 'requestHeaders' might be parsed and passed via 'customHeaders' variable
```

Comment:
What did you mean by unsupported here? content-specific headers?

Reply:
`presetHeaders` is a key in this dictionary https://github.com/dotnet/crank/blob/5fdefc136b45953c3d51d2bfca5930377e2ae2ce/src/Microsoft.Crank.Jobs.Wrk/wrk.yml#L3-L7 but we have our own dictionary `headersDictionary`. If we would want to add more pre-defined headers to our dictionary, that would make sense to pass to GET request, we might want to find another way to pass it to WRK too.

The reason it is a dictionary in the first place and not an array with headers is because it is impossible to pass an array from CLI, the value passes would always be a string. And parsing a string seemed like a complete overkill. And passing string without parsing felt super unsafe 😄